libpod/runtime.go

```diff
@@ -393,6 +393,13 @@ func makeRuntime(runtime *Runtime) (retErr error) {
 	runtime.mergeDBConfig(dbConfig)
 
 	unified, _ := cgroups.IsCgroup2UnifiedMode()
+	if !unified {
+		_, ok := os.LookupEnv("IGNORE_CGROUPSV1_WARNING")
```

Comment:
should we prefix the env var with PODMAN to namesapce it somewhat?

Reply:
I checked with Dan if we need similar for buildah and he agreed. So, I dunno if we need `PODMAN_` and `BUILDAH_` or we go with a generic var name

Reply:
Ah yes if we want it in buildah as well then we can leave it like this I think. I doubt it will cause collisions in the real world.

Reply:
I have pretty strong feelings about namespace pollution. Someone may add this to their `rc` files, then a year later have no recollection what it's for. May I encourage us all to rethink this? I would really prefer `PODMAN_etc`, `BUILDAH_etc`, `SKOPEO_etc`. It's more annoying but for something like this annoying is good.